R/utilities.R

```diff
@@ -936,12 +961,12 @@ collate_ssm_results = function(sample_table,
   if(from_flatfile){
     base_path = config::get("project_base")
     #test if we have permissions for the full gambl + icgc merge
```

Comment:
This needs to be updated. These are referring to the old outputs. NEeds to be switched to the new ones, e.g.

maf_template = config::get("results_flatfiles")$ssm$template$cds$deblacklisted

Reply:
Indeed, good catch!